pyEpiabm/pyEpiabm/core/person.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, microcell, age_group=None):
         self.infection_start_time = None
         self.care_home_resident = False
         self.key_worker = False
+        self.date_positive = None
```

Comment:
We don't have waning immunity now, and maybe you don't need to forward plan too much, but I just wanted to check whether it would be easily posssible to adapt this in future for multiple infections?

Reply:
Date positive will reset when someone first enters the recovered compartment if they were infected, or after 10 days if they were not. So no problems with waning as someone can already test positive on separate occasions.

Reply:
That sounds great!